cmd/cmd.go

```diff
@@ -52,21 +61,35 @@ func Run() int {
 		// TODO(komish): Change output streams
 		configExists = false
 	}
-	if len(os.Args[1:]) < 1 && !configExists {
-		fmt.Printf("Not enough arguments provided and the config file ")
-		fmt.Printf("at path %s was not parsed successfully\n(Error: %s).\n\n", configPath, err)
+	if len(args) < 1 {
+		if showHelp { // Want to see if user wants help first before throwing errors at them
+			PrintHelp()
+			return 0
```

Comment:
I agree, we should absolutely see if the user wants help before hitting them with a stream of errors. 

Let's go ahead and move this evaluation of `showHelp` out of this block (returning this block to how it was) before, and having a standalone evaluation right after we set run the `pflag.Parse()` work. I think it reads a little bit better, and prevents us from checking for configs or doing really anything else if the user has asked for help.

For clarity, I expect the start of `Run()` to look something like this after this adjustment is made.

```go
func Run() int {
    // Arg Parsing Things
    // If showHelp, Print and Exit
    // continue doing other checks.
}
```